spring-boot-docs/src/main/asciidoc/howto.adoc

```diff
@@ -1693,7 +1693,10 @@ properties (with the prefix stripped) when the local `EntityManagerFactory` is c
 
 [[howto-configure-hibernate-naming-strategy]]
 === Configure Hibernate Naming Strategy
-Hibernate defines `Physical` and `Implicit` naming strategies. Spring Boot configures
+Since version 5, Hibernate defines `Physical` and `Implicit` naming strategies which you
```

Comment:
That's even better since the Hibernate docs explain the difference between the old and new naming strategies. I applied your suggestion